conda/meta.yaml

```diff
@@ -30,6 +30,7 @@ test:
     - pytest
     - pytest-cov
     - coverage
+    - datashader==0.7.0
```

Comment:
This doesn't seem right. The test skips without datashader, so it's not required; the things here are those needed to be able to run the tests. You did this because there is no separate "testing environment"?

Reply:
Right. I was trying to run the tests on CI. We should probably set up this repo properly so that there is separate testing env.

Reply:
Can that fit in here? Could be as simple as 
```
conda instal ...
pytest ...
```
in travis file, instead of the conda build line. Since we don't need to auto-upload, makes sense to me.